.gitignore

```diff
@@ -6,3 +6,4 @@ npm-debug.log
 stats.json
 .DS_Store
 fakes3/*
+package-lock.json
```

Comment:
Must be committed! See: https://stackoverflow.com/a/44210813/4628787

Reply:
Alright I have removed that line again.